Library/Formula/protobuf-swift.rb

```diff
@@ -0,0 +1,20 @@
+class ProtobufSwift < Formula
+  homepage "https://github.com/alexeyxo/protobuf-swift"
+  url "https://github.com/alexeyxo/protobuf-swift/archive/1.6.tar.gz"
+  sha256 "1581301212fd8c3aa735e3a9a42444e4f5cb86ac9caf5f477a917f4dfe0eb2e1"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "protobuf"
+
+  def install
+    system "./autogen.sh"
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!